components/Starknet/modules/architecture_and_concepts/pages/Accounts/deploying_new_accounts.adoc

```diff
@@ -1,61 +1,74 @@
 [id="deploying_new_accounts"]
-= Deploying new accounts
+= Deploying a new account
 
-Starknet provides the `deploy_account` transaction in order to deploy new accounts to the
-network.
+The `DEPLOY_ACCOUNT` transaction deploys new accounts to the network.
 
-== Deploy account transaction
+Upon receiving a `DEPLOY_ACCOUNT` transaction, the sequencer performs the following steps:
 
-In order to deploy a new account on StarkNet you need to complete the following steps:
+. Verifies that the address has funds to pay for the deployment.
+. Executes the constructor with the given arguments.
+. Executes the `+__validate_deploy__+` entry point. For more information, see xref:#validate_deploy[].
+. Charges fees from the new account address.
+. Sets the account's nonce to `1`.
 
-* Decide on the account contract that you want to deploy
-* Compute your would-be account address off-chain
-* Send funds to this address
+.Procedure
```

Comment:
I would add to this section a link to the developer tools, and mention that most of them offer an easy way to complete the steps below.